libsrc/_DEVELOPMENT/l/sccz80/9-common/l_pop_ei.asm

```diff
@@ -2,7 +2,10 @@ SECTION code_clib
 SECTION code_l_sccz80
 
 PUBLIC l_pop_ei
+PUBLIC l_pop_ei_jp
```

Comment:
Export this entry point so routines can jump to it and exit through it rather than calling it and then returning.